plumbing/protocol/packp/updreq_decode_test.go

```diff
@@ -287,28 +285,28 @@ func (s *UpdReqDecodeSuite) testDecodeOK(payloads []string) *ReferenceUpdateRequ
 		}
 	}
 
-	r := NewReferenceUpdateRequest()
+	r := NewUpdateRequests()
 	s.Nil(r.Decode(&buf))
 
 	return r
 }
 
-func (s *UpdReqDecodeSuite) testDecodeOkRaw(expected *ReferenceUpdateRequest, raw []byte) {
-	req := NewReferenceUpdateRequest()
+func (s *UpdReqDecodeSuite) testDecodeOkRaw(expected *UpdateRequests, raw []byte) {
+	req := NewUpdateRequests()
 	s.Nil(req.Decode(bytes.NewBuffer(raw)))
-	s.NotNil(req.Packfile)
-	s.compareReaders(req.Packfile, expected.Packfile)
-	req.Packfile = nil
-	expected.Packfile = nil
+	// s.NotNil(req.Packfile)
+	// s.compareReaders(req.Packfile, expected.Packfile)
+	// req.Packfile = nil
+	// expected.Packfile = nil
```

Comment:
If we are leaving this commented, we should add a `TODO` with a comment with the reasoning.